ui/pages/settings/security-tab/security-tab.component.js

```diff
@@ -1025,12 +1118,57 @@ export default class SecurityTab extends PureComponent {
     );
   }
 
+  renderDataCollectionWarning = () => {
+    const { t } = this.context;
+
+    return (
+      <Popover
+        wrapTitle
+        centerTitle
+        onClose={() => this.setState({ showDataCollectionDisclaimer: false })}
+        title={
+          <Icon
+            size={IconSize.Xl}
+            name={IconName.Danger}
+            color={IconColor.warningDefault}
+          />
+        }
+        footer={
+          <Button
+            width={BlockSize.Full}
+            type="primary"
+            onClick={() =>
+              this.setState({ showDataCollectionDisclaimer: false })
+            }
+          >
+            {t('dataCollectionWarningPopoverButton')}
+          </Button>
+        }
+      >
+        <Box
+          display={Display.Flex}
+          flexDirection={FlexDirection.Column}
+          gap={2}
+          margin={4}
+        >
+          <Typography>
```

Comment:
We should not be using Typography, please replace it with the Text component from component-library

Reply:
Done